config.py

```diff
@@ -89,12 +87,12 @@
 optgroup.add_argument("--install-path", help="Path to install to\n"
                                              "INSTALL_PATH from the environment")
 
-# --host
-optgroup.add_argument("--host", help="The host architecture\n"
+# --host-arch
+optgroup.add_argument("--host-arch", help="The host architecture\n"
```

Comment:
May be we rename this as --build-arch?  i assume u meant the host machine architecture you are buidlng this code right?

Reply:
The host/target nomenclature arch comes from the autotools, so we just used what had been established.